.github/workflows/build-and-test.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     - name: Install and test
       run: |
         cd python
-        python -m pip install -U --editable ".[dev,parallel]"
+        python -m pip install -U --editable ".[dev,parallel,numpyro]"
         python -m pytest prophet/tests/
```

Comment:
Ideally we add `--backend "CMDSTANPY" "NUMPYRO"` here too. But the logistic trend model is currently very slow in numpyro, we'll need to fix that first.